docs/src/modules/components/DemoToolbar.js

```diff
@@ -32,12 +32,11 @@ import stylingSolutionMapping from 'docs/src/modules/utils/stylingSolutionMappin
 import codeSandbox from '../sandbox/CodeSandbox';
 import stackBlitz from '../sandbox/StackBlitz';
 
-const Root = styled('div')(({ theme }) => [
+const Root = styled(Box)(({ theme }) => [
```

Comment:
No need for div, sx is built-in in style()

```suggestion
const Root = styled('div')(({ theme }) => [
```

Reply:
🤯 didn't know that ⎯ thanks!